book/website/project-design/pd-overview/pd-overview-repro.md

```diff
@@ -1,44 +1,68 @@
 
 In this page, we highlight some of the most important recommendations for collaboration and communication to ensure that you, and everyone else in your project, understand what the project is about, who the stakeholders are and how they can participate.
 You can visit specific chapters to gain an in-depth understanding and selection of practices that meet the specific requirements in your project.
 
-**Documenting project plans and processes for transparency**:
+## Minimal documentation
 
-Document all proposed plans for the project with information on available resources and recommended practices to ensure everyone is on the same page (literally!).
-Communicate the work culture that you want to promote and policies that ensures the safety and security of both your data and people.
+Your open source project (research, software or hardware) should at a minimum provide a license together with a README file with all the basic, general information a newcomer needs to get oriented.
+It should contain a **recognizable name** for your project, as well as a **declaration of the licensing terms** under which it may be distributed (see the {ref}`OH_licences` chapter).
 
 Documentation should be shared via centralised, findable and accessible platforms.
-It is particularly important to share the vision, mission and milestones clearly. 
+
+## Documenting project plans and processes
+ 
+Document the **current state**, **ongoing development**, and/or any **future plans** for the project.
+Add information on available resources and recommended practices to ensure everyone is on the same page (literally!).
+It may also be important to make it clear what your project is not meant to be.
+
+In addition to the above, it is interesting to document the *principal need* for your project, especially for software and hardware project: what problem are you planning to solve ?
+That includes a **problem description** of whatever issue sparked the project, a **functional description** of how your project is meant to address it, and a **context description** of the users and environment your project is targeting. 
+
+It is particularly important to share the **vision, mission and milestones** clearly. 
 Provide sufficient information for what the expected outcomes and deliverables are.
 
+
 Provide overarching as well as short-term goals and describe expected outcomes to help contributors move away from focusing on a single idea of the feature.
 Describe the possible expansion of features in pre-determined and agreed on ways at stages beyond the initial implementation.
 
-**Project's "Who-is-Who"**:
+## Project's "Who-is-Who"
+
+Include a **list of contributors**, **contribution guidelines**, and information on **contact points** where to ask for help.
+In addition, 
+create a directory to highlight the different stakeholders with their roles in the project, keys skills, interests and contact information (when possible).
```

Comment:
`interests, and contact information`

Reply:
```suggestion
create a directory to highlight the different people and organisation that might get involved, indicate their roles in the project, keys skills, interests, and contact information (when possible).
```

Reply:
It was in the initial text, but would change it anyway.

Reply:
Yeah, best to merge `list of contributors` and `directory of stakeholders` into one. Maybe something like

```
Include a **directory of contributors** with their roles in the project, **contribution guidelines**, and information on **contact points** where to ask for help (when possible).
```

I think contributor skills and interests aren't static enough to warrant placing it in documentation, and contact information is already covered by contact points imho.

Reply:
It is actually a bit different, "stakeholders" also represent people who will/may never "contribute" to the project. Think "doctors who will only look at the picture taken from a flexure microscope" for example. Their needs should be taken into account, but they are not contributors.

"keys skills" refers to skills important for this project, probably quite static.
I would keep it simple here, do you think we should link to https://the-turing-way.netlify.app/collaboration/stakeholder-engagement.html?

Reply:
I asked for more feedback on slack, let's see.

Reply:
We've talked about this in the past Julien and I do agree with reason (1), but as mentioned then, reason (2) is more complicated [than you state](https://www.fasttrackimpact.com/post/why-we-shouldn-t-banish-the-word-stakeholder) here and further discussion on that should happen in a separate issue imho (if necessary), as the word is already being used in The Turing Way as is.

In my view what we need to differentiate is whether we want to document actual people or personas / roles. For contributors we document actual people, but w.r.t. your example about doctors, as the one creating the project documentation they may exist only theoretically to you at that point. How about phrasing it as such:

```
In addition, it helps to create an overview of different possible stakeholder personas and how they may interact with the project.
```

Reply:
```suggestion
- Create an overview of  different putative "personas" {ref}`pd-persona` (people and organisations) and how they may interact with the project. 
```